src/internal/sqs.ts

```diff
@@ -110,6 +110,21 @@ export class SQSClient extends AWSClient {
 
         throw AWSError.parseXML(response.body as string)
     }
+
+    private _buildBody(form: any, request: SendMessageRequestParameters) {
```

Comment:
I would prefer if we could move this logic to the body of the `sendMessage` function. The abstraction cost and cognitive load it introduces aren't worth it, in my humble opinion. 

If we end up performing the same operation in other places at a later point in time, then I would be open to discussing it indeed 👍🏻